multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/HistoricOperationEventGetter.java

```diff
@@ -0,0 +1,26 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.cloudfoundry.multiapps.controller.core.model.HistoricOperationEvent;
+import org.cloudfoundry.multiapps.controller.core.persistence.service.HistoricOperationEventService;
+
+@Named
+public class HistoricOperationEventGetter {
+
+    private final HistoricOperationEventService historicOperationEventService;
+
+    @Inject
+    public HistoricOperationEventGetter(HistoricOperationEventService historicOperationEventService) {
+        this.historicOperationEventService = historicOperationEventService;
+    }
+
+    public List<HistoricOperationEvent> getHistoricOperationEventsByProcessId(String processId) {
```

Comment:
If this is only used in `ProcessStepHelper`, I don't think we need this extra class.
And, if it's only used to check about the `ABORT_EXECUTED` event type, I suggest changing this to 
`historicEventService.createQuery().processId(...).type(ABORT_EXECUTED).list()`
and checking whether this list is empty.

Reply:
Used ProcessHelper instead of this class